include/wx/charts/wxchartgridoptions.h

```diff
@@ -120,6 +120,8 @@ class wxChartGridOptions
     /// @return The options for the vertical grid lines.
     wxChartGridLineOptions& GetVerticalGridLineOptions();
 
+    void SetShowGrid(bool show);
```

Comment:
Let's call this SetShowGridLines to keep it consistent with the other function names.